lib/connector/rjmg16x.yaml

```diff
@@ -0,0 +1,17 @@
+name: RJMG16x
+datasheet: http://www.farnell.com/datasheets/71713.pdf
+description: Shielded 8P8C Modular Connector with Magnetics and LEDs
+ordercodes: [["Farnell", "1357436"]]
+footprint: agg:RJMG16X
+pins:
+- - - [LED1_C, 7, passive]
+    - [LED1_A, 8, passive]
+  - - [LED2_C, 9, passive]
+    - [LED2_A, 10, passive]
+  - - [SHIELD, S, passive]
+- - - [Tct, 1, passive]
+    - [Tx+, 2, passive]
+    - [Tx-, 3, passive]
+    - [Rct, 6, passive]
+    - [Rx+, 4, passive]
+    - [Rx-, 5, passive]
```

Comment:
Pinout is correct but I wonder if it would be more useful with a different orientation, so it can be directly wired to something like the LAN8720. Put the TX/RX pins on the left (makes sense for a connector part, anyway), shield/LEDs on the right. Then line up the TX RX in the same order as the LAN (possibly rearrange the LAN into something more sensible too), and move the centre taps so they're both on the outside or both together rather than one in the middle? I appreciate this was probably inspired by the existing RJHSE-538X which perhaps could afford to be updated too.